packages/notifications/discord/src/index.ts

```diff
@@ -0,0 +1,59 @@
+import type { z } from "zod";
+
+import type {
+  basicMonitorSchema,
+  selectNotificationSchema,
+} from "@openstatus/db/src/schema";
+
+const postToWebhook = async (content: string, webhookUrl: string) => {
+  await fetch(webhookUrl, {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify({
+      content,
+      avatar_url:
+        "https://img.stackshare.io/service/104872/default_dc6948366d9bae553adbb8f51252eafbc5d2043a.jpg",
```

Comment:
Can we add the (optimised, reduced file) `avatar_url` image to the `/public` inside of our repository, so we have it?